exercises/react/src/lib.rs

```diff
@@ -1,12 +1,13 @@
 // Because these are passed without & to some functions,
-// it will probably be necessary for these two types to be Copy.
-pub type CellID = ();
+// it will probably be necessary for these three types to be Copy.
+pub type InputCellID = ();
+pub type ComputeCellID = ();
```

Comment:
It might be worth declaring these as 

```rust
pub struct InputCellID();
pub struct ComputeCellID();
```

to suggest the newtype pattern instead of the typedef pattern. 

Interestingly, I believe that the tests don't care; they should work whether these are type aliases or newtypes. Still, if we're in this for the type safety, we may as well do it right.